src/processes/follow-topic-process.js

```diff
@@ -0,0 +1,72 @@
+const StreamChat = require('stream-chat').StreamChat;
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

Consider destructuring `StreamChat` directly from the require statement for cleaner code.

```diff
- const StreamChat = require('stream-chat').StreamChat;
+ const { StreamChat } = require('stream-chat');
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->